### PR TITLE
Update Dev Documentation

### PR DESCRIPTION
**Pull request by cesnietor**
Updates documentation for local development for both server and web app which was out of date.

### CODE REVIEW

**Review by bexsoft — DISMISSED** [review]
LGTM

**Review by prakashsvmx — APPROVED** [review]

**Review by jinapurapu — DISMISSED** [review]

File: DEVELOPMENT.md [hunk 1]
```diff
@@ -1,3 +1,28 @@
+# Developing Console
+
+Console requires to have the [MinIo](https://github.com/minio/minio) server, Console server and the Console web app running.
```
Comment (kaankabalak) [inline DEVELOPMENT.md:3]: Very minor issue, can we adjust `MinIo` to `MinIO` please?

```suggestion
Console requires to have the [MinIO](https://github.com/minio/minio) server, Console server and the Console web app running.
```

Comment (cesnietor) [reply DEVELOPMENT.md]: @kaankabalak  ah thank you ! done.

**Review by reivaj05 — APPROVED** [review]

**Review by bexsoft — APPROVED** [review]

File: DEVELOPMENT.md [hunk 2]
```diff
@@ -1,3 +1,28 @@
+# Developing Console
+
+Console requires to have the [MinIO](https://github.com/minio/minio) server, Console server and the Console web app running.
```
Comment (allanrogerr) [inline DEVELOPMENT.md:3]:
```suggestion
Console requires to have the [MinIO](https://github.com/minio/minio) Server, MinIO Console Server and the MinIO Console running.
```

**Review by allanrogerr — CHANGES_REQUESTED** [review]

**Pull request merged**